osu.Game/Overlays/BeatmapSet/Scores/ScoreTable.cs

```diff
@@ -0,0 +1,193 @@
+
+                for (int i = 0; i < value.Count; i++)
+                    backgroundFlow.Add(new ScoreTableRowBackground(i));
+
+                Columns = createHeaders(value[0]);
+                Content = value.Select((s, i) => createContent(i, s)).ToArray().ToRectangular();
+            }
+        }
+
+        private TableColumn[] createHeaders(ScoreInfo score)
+        {
+            var columns = new List<TableColumn>
+            {
+                new TableColumn("rank", Anchor.CentreRight, new Dimension(GridSizeMode.AutoSize)),
+                new TableColumn("", Anchor.Centre, new Dimension(GridSizeMode.Absolute, 70)), // grade
+                new TableColumn("score", Anchor.CentreLeft, new Dimension(GridSizeMode.AutoSize)),
+                new TableColumn("accuracy", Anchor.CentreLeft, new Dimension(GridSizeMode.AutoSize)),
+                new TableColumn("player", Anchor.CentreLeft, new Dimension(GridSizeMode.Distributed, minSize: 150)),
+                new TableColumn("max combo", Anchor.CentreLeft, new Dimension(GridSizeMode.Distributed, minSize: 70, maxSize: 90))
+            };
+
+            foreach (var statistic in score.Statistics)
+                columns.Add(new TableColumn(statistic.Key.GetDescription(), Anchor.CentreLeft, new Dimension(GridSizeMode.Distributed, minSize: 50, maxSize: 70)));
+
+            columns.AddRange(new[]
+            {
+                new TableColumn("pp", Anchor.CentreLeft, new Dimension(GridSizeMode.Distributed, minSize: 40, maxSize: 70)),
+                new TableColumn("mods", Anchor.CentreLeft, new Dimension(GridSizeMode.AutoSize)),
+            });
+
+            return columns.ToArray();
+        }
+
+        private Drawable[] createContent(int index, ScoreInfo score)
+        {
+            var content = new List<Drawable>
+            {
+                new OsuSpriteText
+                {
+                    Text = $"#{index + 1}",
+                    Font = OsuFont.GetFont(size: text_size, weight: FontWeight.Bold)
+                },
+                new DrawableRank(score.Rank)
+                {
+                    Size = new Vector2(30, 20)
+                },
+                new OsuSpriteText
+                {
+                    Margin = new MarginPadding { Right = horizontal_inset },
+                    Text = $@"{score.TotalScore:N0}",
+                    Font = OsuFont.GetFont(size: text_size, weight: index == 0 ? FontWeight.Bold : FontWeight.Medium)
+                },
+                new OsuSpriteText
+                {
+                    Margin = new MarginPadding { Right = horizontal_inset },
+                    Text = $@"{score.Accuracy:P2}",
+                    Font = OsuFont.GetFont(size: text_size),
+                    Colour = score.Accuracy == 1 ? highAccuracyColour : Color4.White
+                },
+            };
+
+            var username = new LinkFlowContainer(t => t.Font = OsuFont.GetFont(size: text_size)) { AutoSizeAxes = Axes.Both };
```

Comment:
Might be worth making a `LinkedUsername` class for this (probably other usages around the place?).

Reply:
Idk... Generally it's preceded by some text. E.g. `ParticipantInfo` has `hosted by {username}`, `DirectGridPanel` has `mapped by {username}` (albeit not in a text flow).

Reply:
Just the `AddUserLink` method on `LinkFlowContainer` then?

Reply:
I thought we discussed this yesterday to not do it in this PR. But alright.